utils/openejb-provisionning/src/main/java/org/apache/openejb/resolver/Resolver.java

```diff
@@ -27,14 +27,16 @@
 public class Resolver extends MavenResolver {
     public InputStream resolve(final String rawLocation) {
         final boolean initialized = SystemInstance.isInitialized();
+	final String MVN_JNDI_PREFIX = "mvn:";
```

Comment:
@ShermanMarshall @ShermanMarshall 
That is a variable name and not a constant, therefore: have to match regular expression '^[a-z][a-zA-Z0-9]*$'
I belive:
`final String mvnJNDIPrefix = "mvn:";`
Sounds better.